<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manager</class>
 <widget class="QMainWindow" name="Manager">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PassManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="accounts_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>701</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left; /* position at the top center */
    padding: 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Add, Update, or Delete an Account: </string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>631</width>
       <height>146</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="all">
      <item>
       <layout class="QVBoxLayout" name="lines">
        <item>
         <layout class="QHBoxLayout" name="id"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="acc">
          <item>
           <widget class="QLabel" name="label_acc">
            <property name="text">
             <string>Account:   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_acc"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="user">
          <item>
           <widget class="QLabel" name="label_user">
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_user"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="pass">
          <item>
           <widget class="QLabel" name="label_pas">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_pass">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="texts">
        <item>
         <widget class="QLabel" name="label_comments">
          <property name="text">
           <string>Comments:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_comments"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="buttons">
        <item>
         <widget class="QPushButton" name="pushButton_save">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color:rgb(114, 159, 207);
    border-style: outset;
    border-radius: 10px;
    font: bold 16px;
    min-width: 5em;
    padding: 1px;
}

QPushButton:pressed {
    background-color: rgb(32, 74, 135);
}</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_update">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(115, 210, 22);
    border-style: outset;
    border-radius: 10px;
    font: bold 16px;
    min-width: 5em;
    padding: 1px;
}

QPushButton:pressed {
    background-color: rgb(138, 226, 52);
}</string>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delete">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: rgb(173, 127, 168);
    border-style: outset;
    border-radius: 10px;
    font: bold 16px;
    min-width: 5em;
    padding: 1px;
}

QPushButton:pressed {
    background-color:rgb(92, 53, 102);
}</string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>701</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="accounts_table">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"> QTableView {
alternate-background-color: lightblue;
}

 QTableView::item:focus {
 selection-background-color: blue; 
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_load">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(206, 92, 0);
    border-style: outset;
    border-radius: 10px;
    font: bold 16px;
    min-width: 5em;
    padding: 1px;
}

QPushButton:pressed {
    background-color: rgb(252, 175, 62);
}</string>
       </property>
       <property name="text">
        <string>Load Accounts</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionexit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionexit">
   <property name="text">
    <string>exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
